lms/djangoapps/bulk_email/tasks.py

```diff
@@ -381,7 +382,8 @@ def _get_source_address(course_id, course_title):
     # For the email address, get the course.  Then make sure that it can be used
     # in an email address, by substituting a '_' anywhere a non-(ascii, period, or dash)
     # character appears.
-    from_addr = u'"{0}" Course Staff <{1}-{2}>'.format(
+    # Translators: This is the email subject when the course staff emails the students
```

Comment:
@sarina I'm not sure why I keep mistaking between the two, while it should be really obvious.
Sorry ^_^